app/viewmodels/authorisedOfficials/AuthorisedOfficialsStatusHelper.scala

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.authorisedOfficials
+
+import models.UserAnswers
+import pages.QuestionPage
+import pages.addressLookup._
+import pages.authorisedOfficials._
+import viewmodels.StatusHelper
+
+object AuthorisedOfficialsStatusHelper extends StatusHelper {
+
+  implicit class AuthorisedOfficialStatus(common: Seq[QuestionPage[_]]) {
+
+    def authorisedOfficial1StartOfJourney: Boolean => Seq[QuestionPage[_]] = (isNino: Boolean) => {
+      updateList(
+        isNino,
+        Seq(AuthorisedOfficialsNinoPage(0)),
+        Seq(AuthorisedOfficialsPassportPage(0))
+      )
+    }
+
+    def authorisedOfficial1PreviousAddressEntry: Boolean => Seq[QuestionPage[_]] = (isPreviousAddress: Boolean) => {
+      updateList(
+        isPreviousAddress,
+        Seq(AuthorisedOfficialPreviousAddressLookupPage(0))
+      )
+    }
+
+    def authorisedOfficial2StartOfJourney: Boolean => Seq[QuestionPage[_]] = (isNino: Boolean) => {
+      updateList(
+        isNino,
+        authorisedOfficial2common ++ Seq(AuthorisedOfficialsNinoPage(1)),
+        authorisedOfficial2common ++ Seq(AuthorisedOfficialsPassportPage(1))
+      )
+    }
+
+    def authorisedOfficial2PreviousAddressEntry: Boolean => Seq[QuestionPage[_]] = (isPreviousAddress: Boolean) => {
+      if (isPreviousAddress) common ++ Seq(AuthorisedOfficialPreviousAddressLookupPage(1)) else common
+    }
+
+    def updateList(condition: Boolean, addIfTrue:Seq[QuestionPage[_]], elseAdd : Seq[QuestionPage[_]]= Seq.empty): Seq[QuestionPage[_]] = {
```

Comment:
a bit of scalastyle there - please add a space after, but not before, the colon (`parameter: Type`) and add a space before the =